ovos_classifiers/heuristics/numeric.py

```diff
@@ -2097,3 +2097,571 @@ def _initialize_number_data_az(self, short_scale, speech=True):
         string_num_scale_az = {v: k for k, v in string_num_scale_az.items()}
 
         return multiplies, string_num_ordinal_az, string_num_scale_az
+
+class FrenchNumberParser:
+    # taken from lingua_franca
+    _ARTICLES_FR = {'der', 'das', 'die', 'dem', 'den'}
```

Comment:
@emphasize what will be the right translation for this?

Reply:
you already have that in normalize.json just add those here also